contracts/.gitignore

```diff
@@ -6,6 +6,7 @@ crytic-export/
 .env
 broadcast/
 out/
+binding/src
```

Comment:
@aakoshh When contracts/bindings change and a dev is adjusting the SDK and other downstream components, is the expectation that they'll locally generate the bindings and adapt the dependent code, but only check in the latter and not the former? Then the CI build would generate these _before_ trying to build and would push the updated bindings?

Reply:
This is up for debate, but currently I tried to follow what you said about not checking in generated code at all. So you are expected to update the SDK and Fendermint, check in those changes, but not check in the Ethers generated bindings, and CI will build it but _also not_ push it either, it will never appear on Github.

That said, if it's useful to be able to share links to the Rust code, we can check it in. The reason I didn't do it is because people reported it changes for no reason. 

However today I noticed that `binding/src/lib.rs` changes in a predictable way: when @cryptoAtwill pushes the modules are in alphabetical order; when I push they are in the order of appearance in `build.rs`. So I think there is something that formats the generated code on his end, it's not the fault of `ethers` being different.